caas/kubernetes/provider/specs/v3_test.go

```diff
@@ -1370,7 +1435,7 @@ kubernetesResources:
 `[1:]
 
 	_, err := k8sspecs.ParsePodSpec(specStr)
-	c.Assert(err, gc.ErrorMatches, `ingress name is missing`)
+	c.Assert(err, gc.ErrorMatches, `name is missing`)
```

Comment:
This doesn't seem like a good change - we want to tell the user which name is missing so they know what to fix

Reply:
This change was because we removed the duplicate validation.
`name is missing` was raised from `.Meta`.